terragrunt/github-arc-infra-wg-ci/config.yaml

```diff
@@ -0,0 +1,27 @@
+project: app-runtime-interfaces-wg
+region: europe-west3
+zone: europe-west3-a
+
+gcs_bucket: terraform-wg-ci
+gcs_prefix: github-arc-infra
+
+# Your GKE cluster (assumes one already exists)
+gke_name: wg-ci
+
+arc_github_access_token_name: app-autoscaler-ci-bot-actions-controller-token
+arc_letsencrypt_notifications_email: ApplicationAutoscaler@sap.com
+
+dns_zone: app-runtime-interfaces
+dns_domain: ci.cloudfoundry.org
+
+cert_manager_helm_version: "1.10.1"
+arc_helm_version: "0.21.1"
```

Comment:
Reminder to myself (@silvestre) : We need to add this to renovate.

Reply:
Added in https://github.com/cloudfoundry/app-runtime-interfaces-infrastructure/pull/4/commits/923ed29f2ad0260e36c29572b15f742575b0957a